### PR TITLE
Clean up pull request template and code owners

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1 @@
-packages/custom-elements/ @azakus @bicknellr
-packages/html-imports/ @azakus
-packages/shadycss/ @azakus
-packages/shadydom/ @azakus @sorvell
-packages/template/ @azakus
-packages/url/ @azakus
-packages/webcomponentsjs/ @azakus
```
Comment (bicknellr) [inline .github/CODEOWNERS:7]: I think @dfreedm and @sorvell should weigh in here.

Comment (aomarks) [reply .github/CODEOWNERS:7]: This change just means that we'll be the default automatic reviewers on new PRs, and I figured we'd add in other people to review as needed. Seems like it makes sense for us to do triage in this way. Anybody who wants to get notifications for PRs can just watch the repo. But I'm fine if @dfreedm @sorvell still also want to be assigned to every PR, their call.

Comment (dfreedm) [reply .github/CODEOWNERS:7]: Nope, I'm happy to let the people doing the work be the auto reviewers :)

Comment (sorvell) [reply .github/CODEOWNERS:7]: Same as dfreedm.

**Pull request merged**